### PR TITLE
fix wrong log when do a rollback to a succeed rolling-update

### PR DESCRIPTION
**Pull request by zhangmingld**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
If user did a rollback to a succeed rolling-update RC
They will get the following:
[root@k8s-master:/home/ubuntu/cka]$ kubectl rolling-update rc-rool2 --rollback=true
error: Could not find  to rollback.

after this pr
User will get the following
root@k8s-master:/home/ubuntu/cka]$ kubectl rolling-update rc-rool2 --rollback=true
error: Could not rollback a succeed roling-update or a non-rolling-update RC

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by dixudx — COMMENTED** [review]
/ok-to-test

Please add a test for this to avoid falling back.

**Comment by zhangmingld** [comment]
@dixudx  i am a little confuse , you mean a e2e test ?

File: pkg/kubectl/cmd/rollingupdate/rollingupdate.go [hunk 1]
```diff
@@ -356,6 +356,9 @@ func (o *RollingUpdateOptions) Run() error {
 
 	if o.Rollback {
 		newName := o.FindNewName(oldRc)
+		if len(newName) == 0 {
+			return fmt.Errorf("Could not rollback a succeed roling-update or a non-rolling-update RC")
```
Comment (k-toyoda-pi) [inline pkg/kubectl/cmd/rollingupdate/rollingupdate.go]: Is it correct to change "roling-update" to "rolling-update"?

Comment (zhangmingld) [reply pkg/kubectl/cmd/rollingupdate/rollingupdate.go]: Thankyou , i have changed it

**Pull request closed without merging**